examples/alloydb/backup.yaml

```
@@ -0,0 +1,85 @@
+apiVersion: alloydb.gcp.upbound.io/v1beta1
+kind: Backup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alloydb/v1beta1/backup
+  labels:
+    testing.upbound.io/example-name: default
+  name: default
+spec:
+  forProvider:
+    clusterNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: default
+    location: us-central1
+
+---
+
+apiVersion: alloydb.gcp.upbound.io/v1beta1
+kind: Cluster
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alloydb/v1beta1/backup
+  labels:
+    testing.upbound.io/example-name: default
+  name: default
+spec:
+  forProvider:
+    location: us-central1
+    network: ${data.google_compute_network.default.id}
```

```suggestion
    networkSelector:
      matchLabels:
        testing.upbound.io/example-name: google_compute_network
```

Same here and we should add `Network.compute` resource in this example manifest.